src/components/antdComponents/antdform/AutoCompleteForm.tsx

```diff
@@ -0,0 +1,29 @@
+import { AutoComplete, Form } from 'antd';
+import { useState } from 'react';
+
+const mock = [{ value: 'AMD' }, { value: 'AAPL' }, { value: 'NIO' }, { value: 'SOFI' }, { value: 'DAME' }, { value: 'DAMV' }];
+
+const AutoCompleteForm: React.FC = () => {
```

Comment:
does this need a type? should we be doing this everywhere?

Reply:
I believe this is the way of things --> This is the recommended way on ant d